modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -15,10 +15,41 @@
 from modin.pandas.test.utils import create_test_dfs, test_data_values, df_equals
 from modin.config import NPartitions, Engine
 
+import pandas
 import pytest
 
 NPartitions.put(4)
 
+if Engine.get() == "Ray":
+    import ray
+    from modin.core.execution.ray.implementations.pandas_on_ray.partitioning.partition import (
+        PandasOnRayDataframePartition,
+    )
+    from modin.core.execution.ray.implementations.pandas_on_ray.partitioning.virtual_partition import (
+        PandasOnRayDataframeColumnPartition,
```

Comment:
Why are we only testing column partitions and not row partitions?

Reply:
We should test for row partitions! Thank you for asking this; I found a bug in the implementation for row partitions. I added tests for row partitions.